### ISSUE TITLE
Override class

### ISSUE THREAD

**Issue by liyakat** [Smile-SA/elasticsuite#1916]
is there any possiblity to override below class in my own module ?

 Smile\ElasticsuiteVirtualCategory\Model\Layer\Filter\Category

i have to override  __construct method

Thanks

**Comment by romainruaud** [comment]
Yes you can override any Elasticsuite legacy class as you would do with Magento's legacy classes, through the DI.

Regards
